e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -505,6 +505,7 @@ describe("scenarios > collection defaults", () => {
 
           // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
           cy.findByText(/item(s)? selected/)
+            .parent()
```

Comment:
This test previously worked only because button wasn't actually chained.
We have to properly scope it now after the fix.